<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddConstraintTeachersMaxBuildingChangesPerWeekForm_template</class>
 <widget class="QDialog" name="AddConstraintTeachersMaxBuildingChangesPerWeekForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to add a new constraint of this type">Add teachers max building changes per week</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="addConstraintPushButton">
       <property name="text">
        <string>Add constraint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="weightTextLabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Weight percentage (necessary: 100%)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weightLineEdit">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Max building changes per week</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxChangesSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
